pkg/machine/wsl/machine.go

```diff
@@ -0,0 +1,1055 @@
+// +build amd64 arm64
```

Comment:
The wsl package should only be build for windows. We not need need this code in linux/macos. Just make an empty interface which return an not implemented error for all functions

Reply:
Ah yes I forgot to clean that up. I had it this way for selfish reasons in my local workspace so gopls would function.